packages/core/src/types.ts

```diff
@@ -72,14 +72,14 @@ export type UserOperationPaymasterOverrides<
   TEntryPointVersion extends EntryPointVersion = EntryPointVersion
 > = TEntryPointVersion extends "0.6.0"
   ? {
-      // paymasterAndData overrides only allows '0x' for bypassing paymaster middleware
-      paymasterAndData: EmptyHex;
+      // paymasterAndData overrides allow either '0x' or the data to overwrite for bypassing paymaster middleware
+      paymasterAndData: BytesLike | EmptyHex;
```

Comment:
oh yea interesting... I guess this is fine since we do a `deepHexlify`

I imagine most people will pass in a `Hex`

Reply:
Do you think there's a better way to go about this?

Reply:
There's a Hex type exported from Viem so you can update this to just be Hex instead of the union

Reply:
Addressed in https://github.com/alchemyplatform/aa-sdk/pull/676/commits/3b4e14b473a9649c5dbaf1edcd2eb30637717d8c!